node/tessel-export.js

```diff
@@ -1449,6 +1417,41 @@ function getWifiInfo() {
       });
     }
 
+    // when immediately connecting and restarting the wifi chip, it takes a few moments before an IP address is broadcast to Tessel.
+    // This function keeps checking for that IP until it's available.
+    function recursiveIP(network) {
+      setImmediate(() => {
+        childProcess.exec('ifconfig wlan0', (error, ipResults) => {
```

Comment:
Since `childProcess.exec` is already going to call its callback asynchronously, I don't think this needs to be wrapped in `setImmediate` (unless I've misunderstood something)